images/linux/ubuntu1804.json

```diff
@@ -132,6 +132,14 @@
             ],
             "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
         },
+        {
+            "type": "shell",
+            "inline": [
+                "mkdir -p /etc/skel/.config/configstore",
+                "echo 'export XDG_CONFIG_HOME=$HOME/.config' | tee -a /etc/skel/.bashrc"
+            ],
+            "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+        },
```

Comment:
It would be great to add any comment here, why we add this logic. Link to the issue or something similar

Reply:
Unfortunately, JSON doesn't support comments

Reply:
does it make sense to move these lines to very small shell script? It sounds overkilled but just want to make sure that we will remember the reason.
May be something like:
```
"inline": [
                "# It is my comment",
                "mkdir -p /etc/skel/.config/configstore",
                "echo 'export XDG_CONFIG_HOME=$HOME/.config' | tee -a /etc/skel/.bashrc"
            ],
```

Reply:
Good point, thanks! I've added a comment